config/mbed_lib.json

```diff
@@ -167,6 +167,24 @@
             "SPI_MISO": "p6",
             "SPI_CLK": "p7",
             "SPI_CS": "p8"
+        },
+        "MTB_ADV_WISE_1530": {
+            "SPI_MOSI": "NC",
```

Comment:
Target override is for devices that have SD interface. Any reason to have `NC` here?

Reply:
Ah, this is a workaround for a bug @bridadan is fixing.

At the moment any libraries with multiple storage options (ie storage-selector) will fail to compile if D1/D0 is not defined.

This overload will work as a temporary solution, @bridadan has plans to change the default pins to NC. (@bridadan when did you plan to do that?)

Reply:
...soooon 😄 I hadn't started on that yet, a few things came up. I guess we just need to submit PRs against all of the storage options (we can just use the list in storage-selector).

Reply:
@bridadan - Is it fine to have this fix temporary till you add a proper solution?

Reply:
absolutely!